packages/benchmarks/package.json

```diff
@@ -23,7 +23,6 @@
     "react-jss": "^8.6.1",
     "react-native-web": "^0.11.4",
     "reactxp": "^1.3.3",
-    "styled-components": "link:../styled-components",
```

Comment:
Why did this get removed @kitten?

Reply:
I explicitly removed it, since it's not needed in a monorepo. Yarn Workspaces will create a symlink for `styled-components` (the package; not the repo) in the root `node_modules` folder.